earth2mip/_channel_means.py

```diff
@@ -0,0 +1,237 @@
+channel_means = {
+    "u10m": -0.06854962557554245,
+    "v10m": 0.18743863701820374,
```

Comment:
I suppose we should establish that earth2mip workflows use physical scales and ECMWFs field ids when they exist.

Reply:
if you mean `u10m` vs `u10` the issue I think was that `u10` could be at 10 meter above the ground or at the 10 hap level in the atmosphere (~50km height) 
@nbren12 is that the motivation to change `u10` --> `u10m` ?

Reply:
Let's leave this discussion to another time. Following the pattern of all our other channels (i.e.t850), u10 would be u at 10 hPa, not 10m. This is intentional, and has also been done in all of our recent curated ERA5 datasets.

@NickGeneva  Unlike ECWMF short names, our channels also encode the location in the vertical, so we require someway to disambiguate. The mapping from channel onto ECWMF parameter ID is given [here](https://github.com/NVIDIA/earth2mip/blob/e83c5dbc2badad1d116919ac4a9a6060345446e2/earth2mip/initial_conditions/cds.py#L71). Happy to discuss this more, but there has been some thought put into this issue.

Reply:
Also, the ECWMF short name for this field is 10u, not u10. https://codes.ecmwf.int/grib/param-db/?id=165

Reply:
I dont mind it much what stuff is called, its more about consistency and how we can communicate this to people may want to load from other datasets (e.g. GFS)... how do you convert to our vocab.

E.g. when someone gets a Key error that u10 isnt in the perturbation normalization dictionary, why isnt it? How would they know to switch to u10m?